src/mlpack/core/data/imputation_methods/mode_imputation.hpp

```diff
@@ -0,0 +1,147 @@
+/**
+ * @file mode_imputation.hpp
```

Comment:
Looks like this is part of https://github.com/mlpack/mlpack/pull/1785, so we should remove this one from this PR.

Reply:
Yes I cleaned all the mess now you can refer it at #1790